apps/tsdashboard/wristbandprinter.py

```diff
@@ -65,9 +91,34 @@ def setShowFlags(self, id, first, middle, father, mother, dob, gender,
         self._genderShow = gender
         self._curpShow = curp
 
+#
+# This procedure actually sends the ASCII data to the printer using lpr.
+# First open a temporary file in /tmp
+# Then send a newline and six spaces to the file.
+# Without the spaces, the patient ID is cut off.
+# Then send the wristband string to the file
+# Close the file.
+# The original print to the screen is still present.
+# Then, using the command lpr, print the file to the zebra printer
+#
+# Asumption: The same filename is used each time. The asumption is that lpr will
+# read in the file before another one is created. The printing is very fast, so
+# hopefully this will work. Otherwise, a temporary filename will be needed, and
+# cleanup will have to be done. This way no cleanup is needed.
+#
     def sendToPrinter(self):
+        import sys
+        import os
+
         wristbandStr = self.getUserFormattedWristband()
+        
+        f= open("/tmp/test.txt","w+")
```

Comment:
Do you have code for actually talking to the printer using its API? This sort of thing may not work well when we try and add something like a logo, or qr code.

Reply:
I had a 100% failure trying to use the API. It's something we could play with in the future, but for now ASCII only.